electron/launcher/src/components/TitleBar.js

```diff
@@ -1,30 +1,39 @@
-import React from 'react';
+import React, { useEffect ,useState} from 'react';
 import closeIcon from 'assets/icons/close_icon.svg';
 import minimizeIcon from 'assets/icons/minimize_icon.svg';
 
-const { remote } = window.require('electron')
+const { electron } = window.require('electron')
 
 const TitleBar = props => {
```

Comment:
do we need to comment out TitleBar if you already commented out the place it was used?

Reply:
I think hes saving it for later to add it in.